pkg/testutil/bq.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package testutil provides functions to query bigquery table and upload GCS objects.
+package testutil
```

Comment:
Does this need to be a public package? Or should it go in `internal`?

Reply:
Let's make this internal.

I do find we have duplicated some BQ query logic that's similar to lumberjack. Might worth extracting it out to pkg.

Reply:
+1